try {
    alertt("welcome")
} catch (error) {
    console.log("error");
    
}